docs/walkthrough_collaborate.md

```diff
@@ -2,19 +2,72 @@
 title: Collaborate within eScriptorium
 summary: About features related to collaboration in eScriptorium.
 authors:
-    - Alix Chagué
-date: 2023-01-31
+    - Hugo Scheithauer, Alix Chagué
+date: 2023-02-09
 ---
 
-# Walkthrough : collaborate within eScriptorium
+# Walkthrough: collaborate within eScriptorium
 
-!!! warning
-    The section needs to be completed.
+This walkthrough details features related to collaboration in eScriptorium.
 
-## Create a team or join a team
+## Teams
+
+Users can be grouped by Teams. The main purpose of a team is to facilitate [sharing documents or projects](#share-a-project-or-a-document). Teams are also helpful to identify a group of people working on the same project or coming from the same institution.  
+
+It is possible to manage teams from the [Profile page](walkthrough_users.md). Team management includes:  
```

Comment:
todo: add a link to the subsection of walkthrough_users.md dedicated to the profile page

Reply:
I don't recall how to do it :thinking: is it [Profile page](walkthrough_users.md#Review and edit your profile) without blank spaces?

Reply:
It would be : `[Profile page](walkthrough_users.md#{slug-du-titre}` so: [Profile page](walkthrough_users.md#review-and-edit-your-profile)